.github/workflows/create-release.yml

```diff
@@ -0,0 +1,89 @@
+# Create a stable API NuGet release. This workflow is manually triggered.
+
+name: Create release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  compute-version:
+    uses: ./.github/workflows/gitversion.yml
+    with:
+      prerelease: "false"
+
+  release-flow:
+    needs: compute-version
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          
+      - name: Output nuget version
+        run: |
+          echo Computed nuget version: ${{ needs.compute-version.outputs.package_version }}
+
+      - name: Setup git config
+        run: |
+          # setup the username and email. I tend to use 'GitHub Actions Bot' with no email by default
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          
+      - name: Start release
+        run: |
+          git checkout develop
+          git checkout -b release/${{ needs.compute-version.outputs.package_version }}
```

Comment:
That environment variable is very long and used multiple times. Can we put it in a short-named variable somewhere and use it from there?

Reply:
Done, I use the output as an environment variable now, which is much shorter.